src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -233,7 +233,7 @@ public function testDumpAsFiles()
         $dumper = new PhpDumper($container);
         $dump = print_r($dumper->dump(['as_files' => true, 'file' => __DIR__, 'hot_path_tag' => 'hot']), true);
         if ('\\' === \DIRECTORY_SEPARATOR) {
-            $dump = str_replace('\\\\Fixtures\\\\includes\\\\foo.php', '/Fixtures/includes/foo.php', $dump);
+            $dump = str_replace("'.\\DIRECTORY_SEPARATOR.'", '/', $dump);
```

Comment:
why this replacement here rather than updating the fixture to contain the OS-independent output (which would allow to see that it is actually generated) ?

Reply:
Well, the output is not OS-independent: the paths are.
On Windows, `\` are turned into `DIRECTORY_SEPARATOR`
On Linux, `/` are kept as is (and they work on Windows as you know)